plugins/formal-verification/src/org/jetbrains/kotlin/formver/embeddings/FieldEmbedding.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.embeddings
+
+import org.jetbrains.kotlin.formver.scala.MangledName
+import org.jetbrains.kotlin.formver.scala.silicon.ast.Field
+
+class FieldEmbedding(val name: MangledName, val type: TypeEmbedding) {
+    val field = Field(name, type.type)
```

Comment:
I think this is a good example of where our naming scheme is confusing, this feels like it should be `viperField`.